### PR TITLE
Dev portal/communityupdates

### PR DESCRIPTION
**Pull request by Lorex-ia**

### CODE REVIEW

**Comment by mfahampshire** [comment]
Re-target the PR to be merging into the upcoming release branch ( `release/v1.1.24`) please

**Comment by mfahampshire** [comment]
Also can you remove those 2 commits from myself and Serinko please?

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md [hunk 1]
```diff
@@ -55,7 +55,53 @@ We love seeing our developer community create applications using Nym. If you wou
 
 <!--NymDrive *end-->
 
+<!--NymDashboard *start-->
 
+## <img src='../images/profile_picture/nym_dashboard_pp.svg' style="float: right; width: 75px; height: 75px;">Nym Dashboard
```
Comment (mfahampshire) [inline documentation/dev-portal/src/community-resources/community-applications-and-guides.md:55]: Do they not use the same shield logo as below for the dashboard? nw if they do just checking

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md [hunk 2]
```diff
@@ -55,7 +55,53 @@ We love seeing our developer community create applications using Nym. If you wou
...
+
+## <img src='../images/profile_picture/darkfi_over_nym_pp.png' style="float: right; width: 75px; height: 75px;">DarkFi over Nym
+
+> In this example, DarkFi used Nym's Mixnet to allow users to anonymously connect to other peers in the Darkfi network.
+* [Github](https://github.com/darkrenaissance/darkfi)
+* [Documentation](https://darkrenaissance.github.io/darkfi/clients/nym_outbound.html)
+
+<!--DarkFi *end-->
+
+
```
Comment (mfahampshire) [inline documentation/dev-portal/src/community-resources/community-applications-and-guides.md:95]: Please add this to the list as well https://github.com/dial0ut/nymstr-email

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md [hunk 3]
```diff
@@ -55,7 +55,53 @@ We love seeing our developer community create applications using Nym. If you wou
...
+* [Deployed Website App](https://isnymup.com/)
+
+<!--IsNymUp *end-->
+
+
+<!--DarkFi *start-->
+
+## <img src='../images/profile_picture/darkfi_over_nym_pp.png' style="float: right; width: 75px; height: 75px;">DarkFi over Nym
+
+> In this example, DarkFi used Nym's Mixnet to allow users to anonymously connect to other peers in the Darkfi network.
```
Comment (mfahampshire) [inline documentation/dev-portal/src/community-resources/community-applications-and-guides.md]: This needs to be more specific IMO - DarkFi can now use the mixnet as a pluggable transport for apps such as `ircd`, their p2p version of `irc`

**Comment by mfahampshire** [comment]
I'm rethinking the 'deployed website app' hyperlink text. I think we can change all of them to just 'deployed app' or something less clunkly.

**Comment by mfahampshire** [comment]
also the 'coming soon' for nostr x nym by NTV is no longer true I think - ask cgi / huxian in the community channels for a link to embed there plz

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md (hunk 1 above)
Comment (Lorex-ia) [reply documentation/dev-portal/src/community-resources/community-applications-and-guides.md:55]: resolved this

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md (hunk 2 above)
Comment (Lorex-ia) [reply documentation/dev-portal/src/community-resources/community-applications-and-guides.md:95]: resolved this

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md (hunk 3 above)
Comment (Lorex-ia) [reply documentation/dev-portal/src/community-resources/community-applications-and-guides.md]: resolved this

File: documentation/dev-portal/src/quickstart/overview.md [hunk 4]
```diff
@@ -4,8 +4,7 @@ There are multiple options to quickly connect to Nym and see the network in acti
 
 Demo application:
 * a browser-based 'hello world' [chat application](https://chat-demo.nymtech.net). Either open in two browser windows and send messages to yourself, or share with a friend and send messages to each other through the mixnet!
-
-Proxy traffic with the Nym Socks5 client:
+* a Coconut-scheme based [Credential Library](https://coco-demo.nymtech.net/). This is a WASM implementation of our Coconut libraries which generate raw Coconut credentials. Test it to create and re-randomize your own credentials!
```
Comment (mfahampshire) [inline documentation/dev-portal/src/quickstart/overview.md:7]: Move this to the section above ( demo applications ) - this doesn't use the socks5 client

File: documentation/dev-portal/src/community-resources/community-applications-and-guides.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
+# Community Applications
+
+We love seeing our developer community create applications using Nym. If you would like to share your application with the community, please submit a pull request to the `main` branch of the `nymtech/dev-portal` [repository](https://github.com/nymtech/dev-portal).
+
+<!--Pastenym *start-->
```
Comment (mfahampshire) [inline documentation/dev-portal/src/community-resources/community-applications-and-guides.md]: Can you remove all the `<PROJECT> start` & `<PROJECT> end` comments please?

**Review by mfahampshire — CHANGES_REQUESTED** [review]
A few more changes otherwise looks good to me

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**